oci/tests/BUILD.bazel

```diff
@@ -16,7 +16,12 @@ IMAGES_TO_TEST = {
     genrule(
         name = "pull_{}".format(repo_name),
         outs = [repo_name],
-        cmd = "$(CRANE_BIN) pull {reference} $@ --format=oci --platform={platform}".format(
+        cmd = """
+$(CRANE_BIN) pull {reference} $@ --format=oci --platform={platform}
+cat "$@/index.json" | $(JQ_BIN) --sort-keys -j '.' > "$@/index-tmp.json"
```

Comment:
needs some comments about doing this to make it match the json.encode_indent output that starlark is making

Reply:
not relevant anymore.